Core/src/Geometry/Extent.cpp

```diff
@@ -104,6 +104,17 @@ void Acts::Extent::setEnvelope(const ExtentEnvelope& envelope) {
   m_envelope = envelope;
 }
 
+bool Acts::Extent::contains(const Vector3& vtx) const {
+  Extent checkExtent;
+  for (int ibv = 0; ibv < (int)binValues; ++ibv) {
+    if (constrains((BinningValue)ibv)) {
```

Comment:
Does the static cast not work?

Reply:
It should, I just forgot - will fix it.

Reply:
Changed consistently to loop through `s_binningValues` as we have it exactly for this reason.